OutOfSchool/OutOfSchool.WebApi/Models/ProvidersInfo/AddressProviderInfoDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.WebApi.Models.ProvidersInfo;
+
+public class AddressProviderInfoDto
```

Comment:
As I see the only difference between `AddressProviderInfoDto` and `AddressWorkshopInfoDto` is `CodeficatorAddress` type. Is it expected that mentioned classes will later obtain more differences?
If not - why don't use inheritance, and have one `AddressBaseInfoDto` class without this field and other, derived class, with it?
I'm asking, because I see lot of duplicates, and don't see reasoning for duplications

Reply:
I've fixed this issue. Now  _Address_ and _Codeficatoraddress_  are the same for both  provider and  workshop.

Reply:
Thank you